base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -746,8 +830,10 @@ where T: BlockchainBackend
                 .height -
                 1;
             self.reorganize_chain(fork_height, reorg_chain)?;
+            info!(target: LOG_TARGET, "Chain reorg happened");
```

Comment:
```suggestion
            warn!(target: LOG_TARGET, "Chain has been re-organised! Old tip details height/difficulty. New tip details: block/difficulty");
            
```

Let's add some info to make these logs useful!